app/src/main/java/com/locoquest/app/RetrofitClientInstance.kt

```diff
@@ -6,14 +6,14 @@ import retrofit2.converter.moshi.MoshiConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://geodesy.noaa.gov/api/nde/"
+    private const val BASE_URL = "https://geodesy.noaa.gov/api/nde/"
 
     val retrofitInstance : Retrofit?
         get() {
             // has this object been created yet?
             if (retrofit == null) {
                 // create it!
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
removed redundancy in retrofit call, called just Rertrofit and code still compiles